Gemfile.lock

```diff
@@ -192,7 +192,7 @@ GEM
     i18n (0.9.5)
       concurrent-ruby (~> 1.0)
     ice_nine (0.11.2)
-    image_processing (1.11.0)
+    image_processing (1.12.0)
```

Comment:
### [image_processing](https://github.com/janko/image_processing)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.11.0 → 1.12.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **6** commits. See the full changes on [the compare page](https://github.com/janko/image_processing/compare/v1.11.0...v1.12.0).

These are the individual commits:
- (e8bef4d) [Make #method_missing private, refactor #branch](https://github.com/janko/image_processing/commit/e8bef4dcdb2a4261e83c253584385861c8563beb)
- (0422712) [Add instrumentation support](https://github.com/janko/image_processing/commit/04227125bd35ee3ef2b337824261fc8d44fb3cfc)
- (7eb4034) [Test on JRuby 9.2.13.0](https://github.com/janko/image_processing/commit/7eb4034add8fe1b198d1f1435a6d8f8c151c2374)
- (f5ab15c) [Finish the instrumentation docs](https://github.com/janko/image_processing/commit/f5ab15ce7a74ab6f12e22e4c1cc1ddaa11b14033)
- (d49c59f) [Try testing on libvips 8.10.1](https://github.com/janko/image_processing/commit/d49c59fb558dace8d6c5a0dd4401475405db1636)
- (61d1ad6) [Bump to 1.12.0](https://github.com/janko/image_processing/commit/61d1ad6354fecfea1afcc3ec4a2e1e4b7637d645)


</details>